packages/nodes-base/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"compilerOptions": {
 		"lib": [
+			"dom",
```

Comment:
I think this lib isn't necessary as the node behavior will get executed outside the browser.

Reply:
Hi @MedAliMarz I add it because of this error

```
n8n-nodes-base: ../../node_modules/avsc/types/index.d.ts(149,41): error TS2304: Cannot find name 'Blob'.
```

![image](https://user-images.githubusercontent.com/1489284/124299371-bae02980-db33-11eb-8746-a9afb71638eb.png)


if those errors are not a problem I can remove it, let me know, please :) 


with the `dom` 
![image](https://user-images.githubusercontent.com/1489284/124299668-13afc200-db34-11eb-89e4-46363b0e9660.png)

without the `dom`
![image](https://user-images.githubusercontent.com/1489284/124299742-27f3bf00-db34-11eb-80c1-64710d11a179.png)

Reply:
It seems that the added library works in both server & browser sides that's why it is required to add the dom lib.